### PR TITLE
fix: use ENABLE_FLAGSMITH_REALTIME environment var

### PR DESCRIPTION
**Pull request by kyle-ssg**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [x] I have added information to `docs/` if required so people know about the feature!
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes
ENABLE_FLAGSMITH_REALTIME was setup to determine if the SDK initialised with the realtime boolean. For some reason this was not used in the admin panel.

Demo and production Vercel environments already provide this to the FE.

## How did you test this code?
- Check that ``Project.flagsmithRealtime`` is being set.
image

### CODE REVIEW

**Comment by khvn26** [comment]
Since it's `False` by default, we need to enable it for ECS task definitions.

I'm also renaming the PR title to `fix` so it's visible in the changelog.

**Review by matthewelwell — APPROVED** [review]

**Comment by matthewelwell** [comment]
> Since it's `False` by default, we need to enable it for ECS task definitions.

This env var is read from vercel in our SaaS environments so we just need to make sure it's set there.

**Comment by kyle-ssg** [comment]
Yeah it was all in vercel already

File: infrastructure/aws/production/ecs-task-definition-web.json [hunk 1]
```diff
@@ -167,6 +167,10 @@
                     "name": "TASK_RUN_METHOD",
                     "value": "TASK_PROCESSOR"
                 },
+                {
+                    "name": "ENABLE_FLAGSMITH_REALTIME",
+                    "value": "True"
+                },
```
Comment (kyle-ssg) [inline infrastructure/aws/production/ecs-task-definition-web.json]: I don't think we need this, unless I'm missing a usecase? Config gets to FE via Vercel.

Comment (khvn26) [reply infrastructure/aws/production/ecs-task-definition-web.json]: Sorry, removed!

Comment (matthewelwell) [reply infrastructure/aws/production/ecs-task-definition-web.json]: No, we don't

**Review by khvn26 — APPROVED** [review]

**Pull request merged**